deployment/aws-batch/requirements.txt

```diff
@@ -1 +1 @@
-boto3==1.4.4
+boto3==1.26.132
```

Comment:
This is the requirements file for the little script that updates the batch task definition. Long-term we want to get rid of it and handle the task definition in Terraform, but for now we're still using it. This upgrade was necessary because the very-old version of boto3 didn't know about the `linuxParameters` option.